README.md

```diff
@@ -57,3 +57,9 @@ If you run into problems while installing the page type, please also refer to th
 page in the [Pageflow  repository](https://github.com/codevise/pageflow). If that 
 doesn't help, consider 
 [filing an issue](https://github.com/codevise/pageflow-external-links/issues).
+
+---
+
+## Contributing locales to pageflow-external-links
+
+- Edit the translations directly on the [pageflow-external-links](http://www.localeapp.com/projects/public?search=pageflow-external-links) project on Locale.
```

Comment:
Maybe also note that changes will be pulled in by the maintainers regulary. Once should probably also add a wiki page to the main pageflow repository which describes the process especially for adding new translation keys in pull requests.